docs/src/componentDocs/Chip/examples/Chip.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {Box} from '@mui/material';
+import {ChipExample} from './ChipExample';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+
+const codeSnippet = `
+<Chip>Outlined Chip</Chip>
```

Comment:
Snippets are adding an extra blank line at the top, please move `<Chip>Outlined Chip</Chip>` at the first line